shared-data/command/types/setup.ts

```diff
@@ -337,3 +350,9 @@ interface LoadLidResult {
   definition: LabwareDefinition
   locationSequence?: LabwareLocationSequence
 }
+
+export interface SetTipStateParams {
+  labwareId: string
+  wellNames: string[]
+  tipWellState?: 'clean' | 'used' | 'empty'
```

Comment:
small nit: i recommend turning this into a type that is exportable, in case another place needs to import it.

```suggestion
  tipWellState?: TipWellState
```

then up above 

```
export type TipWellState = 'clean' | 'used' | 'empty'
```